crux-http-server/src/crux/http_server.clj

```diff
@@ -159,6 +159,21 @@
       (.close ctx)
       (throw t))))
 
+(defn- streamed-iterator-response [^TxLogIterator iterator]
```

Comment:
there's enough commonality here with `streamed-edn-response` to justify re-using it, IMO
commonality is piped-input-stream, writing out an edn sequence, returning the response - so that should be common
difference is in how to get hold of the edn sequence, what to close - so that should be in the callers

I can put in a commit for this one :)